src/Dependencies/VisualStudioText/project.json

```diff
@@ -0,0 +1,12 @@
+{
+  "supports": { },
+  "dependencies": {
+    "Microsoft.VisualStudio.Text.Data": "14.1.24720",
+    "Microsoft.VisualStudio.Text.Logic": "14.1.24720",
+    "Microsoft.VisualStudio.Text.UI": "14.1.24720",
+    "Microsoft.VisualStudio.Text.UI.Wpf": "14.1.24720"
```

Comment:
Should we just merge this with your VisualStudioEditor dependency? I admit that if I was asked which of these two a new reference should go into, I wouldn't have a clear answer.